### PR TITLE
wallet apps update

### PR DESCRIPTION
**Pull request by Jibz-Mysten**
## Description 

Describe the changes or additions included in this PR.

- Switch to tailwind and remove scss for connected apps
- Clicking on the App in the all apps section take you to the app site
- UI fixes

Screenshot 2023-06-22 at 11 40 16 AM
Screenshot 2023-06-22 at 11 39 50 AM
Screenshot 2023-06-22 at 11 39 00 AM

Screenshot 2023-06-22 at 11 38 53 AM

### CODE REVIEW

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 1]
```diff
@@ -19,80 +18,123 @@ export type DAppEntry = {
...
+	name: string;
+	link: string;
+	icon?: string;
+};
+
+function CardView({ name, link, icon }: CardViewProp) {
 	const appUrl = getDAppUrl(link);
 	const originLabel = appUrl.hostname;
+	return (
+		<div className="bg-white group flex flex-col p-3.75 box-border w-full rounded-2xl border border-gray-45 border-solid h-32 hover:bg-sui/10 hover:border-sui/30">
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:32]: Where is `group` used?

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 2]
```diff
@@ -19,80 +18,123 @@ export type DAppEntry = {
...
+	icon?: string;
+	description: string;
+	tags?: string[];
+};
 
-				{displayType === 'full' && tags?.length && (
-					<div className={st.tags}>
+function ListView({ name, icon, description, tags }: ListViewProp) {
+	return (
+		<div className="bg-white group flex py-3.5 px-1.25 gap-3 item-center box-border rounded hover:bg-sui/10">
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:60]: same here

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 3]
```diff
@@ -19,80 +18,123 @@ export type DAppEntry = {
...
+	link,
+	icon,
+	tags,
+	permissionID,
+	displayType,
+}: SuiAppProps) {
+	const [showDisconnectApp, setShowDisconnectApp] = useState(false);
+	const appUrl = getDAppUrl(link);
+
+	if (permissionID && showDisconnectApp) {
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:110]: We need a way to go to the DisconnectApp view when user is in connected apps page

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 4]
```diff
@@ -19,80 +18,142 @@ export type DAppEntry = {
...
+				link={link}
+				icon={icon}
+				permissionID={permissionID}
+				setShowDisconnectApp={setShowDisconnectApp}
+			/>
+		);
+	}
+
+	const AppDetails = (
 		<>
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:125]: the fragment is not necessary here, just do something like 
```
displayType === 'full' ? <List.... : <Card...
```

File: apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx [hunk 5]
```diff
@@ -19,80 +18,142 @@ export type DAppEntry = {
...
-					{AppDetails}
-				</ExternalLink>
+				<CardView name={name} link={link} icon={icon} />
 			)}
 		</>
 	);
+
+	if (permissionID && !openAppSite) {
+		return (
+			<button
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:136]: We could use a div here, right?

Comment (Jibz-Mysten) [reply apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:136]: I use button for accessibility, I will change it

Comment (pchrysochoidis) [reply apps/wallet/src/ui/app/components/sui-apps/SuiApp.tsx:136]: Yeah we could add `role="button"`

File: apps/wallet/src/ui/app/components/sui-apps/SuiAppEmpty.tsx [hunk 6]
```diff
@@ -1,27 +1,38 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import cl from 'classnames';
+import { cva, type VariantProps } from 'class-variance-authority';
 
-import { type DisplayType } from './SuiApp';
+const appEmptyStyle = cva(['flex gap-3 gap-3 p-3.75 h-28'], {
```
Comment (pchrysochoidis) [inline apps/wallet/src/ui/app/components/sui-apps/SuiAppEmpty.tsx:6]: Remove one of the `gap-3`

**Review by pchrysochoidis — APPROVED** [review]

**Pull request merged**